Packs/OktaITAdmin/Integrations/Okta_IT_Admin/Okta_IT_Admin.py

```diff
@@ -0,0 +1,789 @@
+            "brand": self.brand,
+            "instanceName": self.instanceName,
+            "success": success,
+            "active": active,
+            "id": iden,
+            "username": username,
+            "email": email,
+            "errorCode": errorCode,
+            "errorMessage": errorMessage,
+            "details": details
+        }
+
+
+'''HELPER FUNCTIONS'''
+
+
+def verify_and_load_scim_data(scim):
+    try:
+        scim = json.loads(scim)
+    except Exception:
+        pass
+    if type(scim) != dict:
+        raise Exception("SCIM data is not a valid JSON")
+    return scim
+
+
+def map_scim(scim):
+    try:
+        scim = json.loads(scim)
+    except Exception:
+        pass
+    if type(scim) != dict:
+        raise Exception('Provided client data is not JSON compatible')
+
+    scim_extension = SCIM_EXTENSION_SCHEMA.replace('.', '\.')
+
+    mapping = {
+        "active": "active",
+        "countryCode": "addresses(val.primary && val.primary==true).[0].country",
+        "addressFormatted": "addresses(val.primary && val.primary==true).[0].formatted",
+        "city": "addresses(val.primary && val.primary==true).[0].locality",
+        "zipCode": "addresses(val.primary && val.primary==true).[0].postalCode",
+        "state": "addresses(val.primary && val.primary==true).[0].region",
+        "streetAddress": "addresses(val.primary && val.primary==true).[0].streetAddress",
+        "addressType": "addresses(val.primary && val.primary==true).[0].type",
+        "costCenter": scim_extension + ".costcenter",
+        "department": scim_extension + ".department",
+        "displayName": "displayName",
+        "division": scim_extension + ".division",
+        "email": "emails(val.primary && val.primary==true).[0].value",
+        "emailType": "emails(val.primary && val.primary==true).[0].type",
+        "mobilePhone": "phoneNumbers(val.type && val.type=='mobile').[0].value",
+        "employeeNumber": scim_extension + ".employeeNumber",
+        "groups": "groups(val.display).display",
+        "id": "id",
+        "externalId": "externalId",
+        "locale": "locale",
+        "manager": scim_extension + ".manager.value",
+        "lastName": "name.familyName",
+        "nameFormatted": "name.formatted",
+        "firstName": "name.givenName",
+        "nameHonorificPrefix": "name.honorificPrefix",
+        "nameHonorificSuffix": "name.honorificSuffix",
+        "nameMiddleName": "name.middleName",
+        "nickName": "nickName",
+        "organization": scim_extension + ".organization",
+        "password": "password",
+        "photo": "photos(val.type && val.type=='photo').[0].value",
+        "preferredLanguage": "preferredLanguage",
+        "profileUrl": "profileUrl",
+        "thumbnnail": "photos(val.type && val.type=='thumbnail').[0].value",
+        "timezone": "timezone",
+        "title": "title",
+        "userName": "userName",
+        "login": "userName",
+        "userType": "userType"
+    }
```

Comment:
do we need all these? (same comment as in Slack IT Admin)

Reply:
removed unused keys